src/datadog.ts

```diff
@@ -90,6 +103,14 @@ export class Datadog extends cdk.Construct {
       this.transport.applyEnvVars(lambdaFunctions);
     }
   }
+
+  public addForwarderToLogGroups(logGroups: logs.LogGroup[]) {
+    if (this.props.forwarderArn !== undefined) {
+      addForwarderToLogGroup(this.scope, logGroups, this.props.forwarderArn);
```

Comment:
Nit: I feel like its strange that we have a `addForwarderToLogGroups` (plural) and a `addForwarderToLogGroup` (singular) but they both can take in multiple log groups. I feel like the call to `addForwarderToLogGroup` (line 109) is a bit misleading maybe we can think of a different name. 

I know the user wont ever know that we call another function inside the `addForwarderToLogGroups` function but it may help with code readability when developers look back at this.